src/Benchmarks/EventPerf.cs

```diff
@@ -0,0 +1,107 @@
+﻿using BenchmarkComponent;
+using BenchmarkDotNet.Attributes;
+using System;
+
+namespace Benchmarks
+{
+    [MemoryDiagnoser]
+    public class EventPerf
+    {
+        ClassWithMarshalingRoutines instance;
+        int z2;
+        ClassWithMarshalingRoutines instance2;
+        int z4;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            instance = new ClassWithMarshalingRoutines();
+            System.EventHandler<int> s = (object sender, int value) => z2 = value;
+            instance.IntPropertyChanged += s;
+
+            instance2 = new ClassWithMarshalingRoutines();
+            System.EventHandler<int> s2 = (object sender, int value) =>
+            {
+                if (sender == this)
+                    z4 = value;
+                else
+                    z4 = value * 3;
+            };
+            instance2.IntPropertyChanged += s2;
+        }
+
+        [Benchmark]
+        public object IntEventOverhead()

```

Comment:
this is measuring construction time of ClassWithMarshalingRoutines and System.EventHandler - how come?

Reply:
This was so that I can actually calculate the time it takes to add an event.  The diff between AddIntEventToNewEventSource and this would be that.  This essentially calculates the overhead in that other benchmark.